doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -309,19 +309,13 @@ You can schedule actors or tasks to a placement group using
 
   When you use an actor with a placement group, always specify ``num_cpus``.
 
-  When you don't specify (e.g., ``num_cpus=0``), a placement group option is ignored,
-  and the task and actor don't use the reserved resources.
-  
-  Note that by default (with no arguments to ``ray.remote``),
-
-  - Ray task requires 1 CPU
-  - Ray actor requires 1 CPU when it is scheduled. But after it is created, it occupies 0 CPU.
+  By default (with no arguments to ``ray.remote``) Ray actor requires 1 CPU when it is scheduled. But after it is created, it occupies 0 CPU.
 
   When scheduling an actor without resource requirements and a placement group, the placement group has to be created (since it requires 1 CPU to be scheduled).
```

Comment:
```suggestion
  When scheduling an actor with no resource requirements and a placement group, the placement group gets created (since the actor requires 1 CPU to be scheduled).
```